docs/index.yaml

```diff
@@ -1,6 +1,29 @@
 apiVersion: v1
 entries:
   hpe-csi-driver:
+  - apiVersion: v1
+    appVersion: 1.4.0-beta
+    created: "2020-12-04T12:11:08.980181-08:00"
+    description: A Helm chart for installing the HPE CSI Driver for Kubernetes
+    digest: 00ec485d2d43324d9875930698bc1695ebaded15f4abbdb805dde8070aca5bc5
+    home: https://hpe.com/storage/containers
+    icon: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
+    keywords:
+    - HPE
+    - Storage
+    - StorageClass
+    - CentOS
+    - Ubuntu
+    - RHEL
+    maintainers:
+    - email: hpe-containers-dev@hpe.com
+      name: raunakkumar
```

Comment:
We may need to use `hpe-containers-dev` instead of developer name. Please check with @datamattsson